corehq/apps/notifications/views.py

```diff
@@ -20,17 +29,65 @@ def get(self, request, *args, **kwargs):
 
     @allow_remote_invocation
     def get_notifications(self, in_data):
-        # todo actual models
         # todo always grab alerts if they are still relevant
```

Comment:
@biyeun do you remember what you meant by "relevant" in this comment? I think it can be deleted